tests/containers/skopeo.pm

```diff
@@ -0,0 +1,104 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: skopeo
+# Summary: Test basic skopeo commands.
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';    # used in select_serial_terminal
+use utils 'zypper_call';    # used in zypper_call
+use version_utils 'package_version_cmp';
+use version_utils 'is_transactional';
+use transactional;
+use containers::common qw(install_packages);
+
+sub run {
+    my ($self, $args) = @_;
+
+    # Required packages
+    my @packages = qw(skopeo jq);
+
+    select_serial_terminal();    # Select most suitable text console
+
+    # Set a variable for my remote image
+    my $remote_image = 'registry.suse.com/bci/bci-busybox:latest';
+    # Set a variable for my working directory
+    my $workdir = '/tmp/test';
+
+    # install_packages accounts for SLE-Micro environment with transactional-update
+    record_info('Installing packages', 'Install required packages');
+    install_packages(@packages);
+
+    # Verify that skopeo version emits a reasonable output.
+    record_info('Verify version', 'Verify that skopeo version is greater or equal to 1.12.0.');
+    die('wrong skopeo version') unless (package_version_cmp('1.12.0', script_output('skopeo --version | cut -d" " -f3')) <= 0);
```

Comment:
I am a bit unsure about this check. Is out there a version that has problems with its output? If yes, is there a bug?